exercises/triangle/triangle_test.py

```diff
@@ -1,67 +1,65 @@
 import unittest
 
-from triangle import Triangle, TriangleError
+from triangle import equilateral, isosceles, scalene
 
 
-class TriangleTests(unittest.TestCase):
-    def test_equilateral_triangles_have_equal_sides(self):
-        self.assertEqual(Triangle(2, 2, 2).kind(), "equilateral")
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

Comment:
Looks like these are actually the tests from version 1.1.0 now - there was an update in `problem-specifications` that moved inputs into an `input` key.